modules/rostests/apitests/user32/GetMessageTime.c

```diff
@@ -47,6 +47,8 @@ WindowProc(HWND hwnd, UINT uMsg, WPARAM wParam, LPARAM lParam)
             {
                 ok(GetMessageTime() - s_nMsgTime >= TIMER_INTERVAL / 2,
                    "message time is wrong\n");
+                ok(GetTickCount() - (DWORD)GetMessageTime() < TIMER_INTERVAL / 2,
+                   "message time is wrong\n");
```

Comment:
May I suggest "GetMessageTime() is wrong, compared to GetTickCount()\n", to be more explicit? Everywhere.

Reply:
Thanks!